editor/src/core/shared/dom-utils.ts

```diff
@@ -204,6 +204,8 @@ export const intrinsicHTMLElementNamesThatSupportChildren: Array<string> = [
   'span',
 ]
 
+export const textElements = ['text', 'p', 'span', 'h1', 'h2', 'h2', 'h3', 'h5', 'h6']
```

Comment:
ultra minor nitpick, but we use PascalCase for global consts 
```suggestion
export const TextElements = ['text', 'p', 'span', 'h1', 'h2', 'h2', 'h3', 'h5', 'h6']
```